### ISSUE TITLE
[BUG] documentation update for OpenWRT.

### ISSUE THREAD

**Issue by TommyV6** [linuxserver/docker-netbootxyz#62]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The command provided for OpenWRT does not work on OpenWRT 21.02.0 or latest.

### Expected Behavior

Inform host here is netboot.xyz

### Steps To Reproduce

Type in provided commands for OpenWRT. 

### Environment

```markdown
- OS: OpenWRT 23.05.2
Docker service works.
```

### CPU architecture

arm64

### Docker creation

```bash
docker run -d \
  --name=netbootxyz \
  -e PUID=1000 \
  -e PGID=1000 \
  -e TZ=Etc/UTC \
  -e MENU_VERSION=1.9.9 `#optional` \
  -e PORT_RANGE=30000:30010 `#optional` \
  -e SUBFOLDER=/ `#optional` \
  -p 3000:3000 \
  -p 69:69/udp \
  -p 8080:80 `#optional` \
  -v /path/to/config:/config \
  -v /path/to/assets:/assets `#optional` \
  --restart unless-stopped \
  lscr.io/linuxserver/netbootxyz:latest
```

### Container logs

```bash
none.
```

**Comment by j0nnymoe** [comment]
If it's not working on openwrt, it would be helpful if you can provide the error that makes you think it's not working.

**Comment by dezeroku** [comment]
When the current instructions are applied, only the last setting persists in the config, effectively overriding the previous ones.

There are few threads about the issue on OpenWRT forum and the consensus seems to be to write to a dnsmasq confg file, either directly to `/etc/dnsmasq.conf` or to [a separate file, that's later included in the config](https://forum.openwrt.org/t/does-dnsmasq-support-client-arch-code-93/138709/4?u=dezeroku).

TL;DR
1. Add the below entries to `/etc/dnsmasq.conf`
```
dhcp-match=set:bios,60,PXEClient:Arch:00000
dhcp-boot=tag:bios,netboot.xyz.kpxe,,YOURSERVERIP
dhcp-match=set:efi32,60,PXEClient:Arch:00002
dhcp-boot=tag:efi32,netboot.xyz.efi,,YOURSERVERIP
dhcp-match=set:efi32-1,60,PXEClient:Arch:00006
dhcp-boot=tag:efi32-1,netboot.xyz.efi,,YOURSERVERIP
dhcp-match=set:efi64,60,PXEClient:Arch:00007
dhcp-boot=tag:efi64,netboot.xyz.efi,,YOURSERVERIP
dhcp-match=set:efi64-1,60,PXEClient:Arch:00008
dhcp-boot=tag:efi64-1,netboot.xyz.efi,,YOURSERVERIP
dhcp-match=set:efi64-2,60,PXEClient:Arch:00009
dhcp-boot=tag:efi64-2,netboot.xyz.efi,,YOURSERVERIP
```
2.  Run `/etc/init.d/dnsmasq restart` to apply the changes

**Comment by thelamer** [comment]
I don't currently run openwrt, but if this works then this should be updated in the readme-vars.yaml file
